mobile.js

```diff
@@ -32,4 +80,117 @@ document.documentElement.addEventListener('click', (e) => {
     hamburguerIcon.classList.remove('hide');
     closeIcon.classList.remove('show');
   }
-});
\ No newline at end of file
+  if (e.target.matches('.see-project')) {
+    const modalWindow = document.querySelector('.modal-window');
+    modalWindow.style.display = 'block';
+  }
+  if (e.target.matches('.close-xbtn')) {
+    const modalWindow = document.querySelector('.modal-window');
+    modalWindow.style.display = 'none';
+  }
+  if (e.target.matches('.ic_menu *')) {
+    hamburguerIcon.classList.toggle('hide');
+    closeIcon.classList.toggle('show');
+    menuIcon.classList.toggle('hide');
+  }
+});
+
+/// // SEE-THIS-PROJECT///
+
+function showModal() {
+  const modalWindow = document.querySelector('.body');
+  const worksHtml = `<div class="modal-window px-2">
+<div class="pop-work-card">
+<div class="pop-header p-0 d-flex flex-column flex-md-row-reverse w-100">
+<button class="close-xbtn fa-solid fa-xmark align-self-end m-0"></button>
+<div class="title m-0 p-0">${projects[0].name}</div>
+</div>
+
+    <div class="pop-tech-container m-0 mt-3">
+    <ul class="ps-0 d-flex gap-1" >
+    <li class="pop-language1 sm w-100">${projects[0].languages[0]}</li>
+    <li class="pop-language2 sm w-100">${projects[0].languages[1]}</li>
+    <li class="pop-language3 sm w-100">${projects[0].languages[2]}</li>
+    </ul>
+    </div>
+    <img class="pop-img1 p-0 w-100" src=${projects[0].image} alt="">
+    
+      <div class="mx-0 modal-text-cont mt-4" >
+        <p class="pop-text mx-auto p-0">${projects[0].description}</p>
+  
+        <div class="live-source-btns mx-auto"> 
+          <a href=${projects[0].linkdemo} class="see-live">See Live</a>
+          <a href=${projects[0].github} class="see-source">See Source</a>
+          </div> 
+      </div>
+      
+      <div class="pop-desktop-btns">
```

Comment:
- [ ] You have done so great to ensure that when the user clicks on the `see project` the popup is displayed and also when clicking on the `close` button the popup disappears 👏 👍 . However, there are a few changes remaining to implement so that our portfolio is close to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#project-requirements) :
     - I have noticed that when the user clicks on the button, the `project detailed#1` is displayed instead of dynamically displaying the details for that particular project. Kindly ensure that when the user clicks on the `see project`, the details of that particular project are displayed.
     - In addition, kindly ensure that there is no horizontal scroll when the popup modal is displayed.  The experience of scrolling horizontally can make the screen judder and create a headache-inducing feeling 😞 . Please consider this to ensure responsiveness in both and all desktop and mobile screen sizes ☺️ .

Video 😎 . The project numbers do not reflect your project. However, they give you an overview of how the targetted project is not displayed.

https://user-images.githubusercontent.com/99394374/224477295-ebdd3fea-5f8d-4405-a033-74e88db21205.mp4

Reply:
@AlphaNtihinduka Thank you for your feedback! ok We've fixed these problems!